management-core/pom.xml

```diff
@@ -10,7 +10,6 @@
     <modelVersion>4.0.0</modelVersion>
     <artifactId>management-core</artifactId>
     <properties>
-        <mysql-connector-java.version>8.0.12</mysql-connector-java.version>
```

Comment:
Why did you remove this?

Reply:
When I introduce my custom exception, I was returning to commons DataIntegrity.. which is in that package so by mistake I was doing some test in order to share that import with common project and I just forget to move back this property, because at the end I not need this reference anymore but in the core module